client/src/components/History/History.js

```diff
@@ -8,27 +8,29 @@ import {
 } from 'react-router-dom';
 import { useSelector, useDispatch } from 'react-redux';
 import { Button, Nav, NavItem, NavLink } from 'reactstrap';
-import { string } from 'prop-types';
 import queryString from 'query-string';
 
+import { Section } from '_common';
 import JobHistory from './HistoryViews';
 import JobHistoryModal from './HistoryViews/JobHistoryModal';
 import * as ROUTES from '../../constants/routes';
 import HistoryBadge from './HistoryBadge';
+
+import './History.global.css';
 import './History.module.scss';
 
 const root = `${ROUTES.WORKBENCH}${ROUTES.HISTORY}`;
 
-const Header = ({ title }) => {
+const Actions = () => {
```

Comment:
__Explanation__
Renamed because:
- [the `title` is now passed directly to `<Section header={…}>`](https://github.com/TACC/Core-Portal/pull/350/files#diff-822e593d04ae92f675a0720dcedb146521e54c31c0517d14937f0f02bf0ade98L149)
- [the `<Actions>` is passed to `<Section headerActions={…}>`](https://github.com/TACC/Core-Portal/pull/350/files#diff-822e593d04ae92f675a0720dcedb146521e54c31c0517d14937f0f02bf0ade98R155)